File: Part1_강의자료/Ch04_전체탐색_시뮬레이션/문제별코드/10250_ACM호텔/10250.java
import java.util.Scanner;

class Main
{
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);
        int T = sc.nextInt();
        while (T-- > 0) {
            int H = sc.nextInt();
            int W = sc.nextInt();
            int N = sc.nextInt();

            int floor = ((N - 1) % H) + 1;  // [1, H]
            int number = (N - 1) / H + 1; // [1, W]
            System.out.printf("%d%02d\n", floor, number);
        }
    }
}

File: Part1_강의자료/Ch04_전체탐색_시뮬레이션/문제별코드/10250_ACM호텔/10250.cpp
#include <stdio.h>

int T, H, W, N;

int main()
{
    scanf("%d", &T);
    while (T--)
    {
        scanf("%d %d %d", &H, &W, &N);

        int floor = (N - 1) % H + 1;
        int number = (N - 1) / H + 1;

        printf("%d%02d\n", floor, number);
    }
    return 0;
}
